doc/source/whatsnew/v0.23.5.txt

```diff
@@ -49,3 +49,5 @@ Bug Fixes
 **I/O**
 
 - Bug in :func:`read_csv` that caused it to raise ``OverflowError`` when trying to use 'inf' as ``na_value`` with integer index column (:issue:`17128`)
+- Bug in :func:`to_string(index=False)` that broke column alignment (:issue:`16839`, :issue:`13032`)
```

Comment:
move to 0.24.0

can you make this more explicit, e.g. say what cases it is fixing.